elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -0,0 +1,115 @@
+
+ul.connectedSortable {
```

Comment:
All other CSS classes are using kebab-case except for this one. Could we change it to `connected-sortable` for consistency? Actually it should be `pl-order-blocks-connected-sortable` to namespace it.

Reply:
fixed. variable renamed to your suggested name.